frontend/src/app/application-forms/validators/currency-validation.ts

```diff
@@ -0,0 +1,14 @@
+import { AbstractControl, NG_VALIDATORS, Validator, ValidatorFn, Validators } from '@angular/forms';
+
+export function currencyValidator(): ValidatorFn {
```

Comment:
Kinda surprising that there isn't an existing validator for this. Or at least is there a `float` validator that you could set for this?